nautobot/docs/user-guide/platform-functionality/savedview.md

```diff
@@ -43,3 +44,123 @@ The `config` attribute holds the configuration of the list view, stored as a `di
+
+Congratulations, you have successfully updated your saved view!
+
+![Updated saved view](../feature-guides/images/saved-views/updated-saved-view.png)
+
+### How to navigate Saved Views
+
+There are several ways you can navigate saved views. All saved views created by any user for any list view can be accessed from the navigation menu on the left hand side under "Extensibility -> Saved Views" as an admin or user with "users:view_savedview" permission:
+
+![Navigation Menu](../feature-guides/images/saved-views/navigation-menu.png)
+
+Another way is from the "Saved View" dropdown button on any given model list view. All saved views defined for this specific model list view are populated in the dropdown menu and the saved view the user is currently viewing will be highlighted in bold.
+
+![Location List View with Saved Views](../feature-guides/images/saved-views/location-list-view-with-saved-views.png)
+
+![Current Saved View](../feature-guides/images/saved-views/current-saved-view-drop-down-menu.png)
+
+### How to set a saved view as global default for all users
+
+There can only be one global default for each object list view and the only way to set a saved view as the global default for all users is from the SavedViewTable at SavedViewListView that is only accessible to admins and users with "users:view_savedview" and "users:edit_savedview" permissions.
+
+![Saved View Table Edit Button](../feature-guides/images/saved-views/saved-view-admin-edit-buttons.png)
+
+Clicking on the edit button will direct you to a standard edit view where you can edit the `name`, `is_shared` and `is_global_default` attributes of the current saved view.
+
+![Saved View Edit View](../feature-guides/images/saved-views/saved-view-admin-edit-view.png)
+
+Check the box for `is_global_default` and click on the Update button.
+
+You should be redirected to the SavedViewListView with a success message. Now all users without a default view of their own will be directed to "Campus Staing Location List View" when they are trying to access location list view.
+
+![Saved View Edit Success](../feature-guides/images/saved-views/saved-view-admin-edit-success.png)
+
+### How to hide a saved view from all other users but yourself
+
+In order to create a saved view only for yourself to see, you should make sure that you uncheck the box for `is_shared` when you are creating a new saved view. This will make sure that you are the only who is able to see this saved view.
+
+![Saved View Modal Unchecked](../feature-guides/images/saved-views/saved-view-modal-unchecked.png)
+
+As an admin or a user with the appropriate permissions, you can also edit the `is_shared` attribute from Saved View Edit View. Make sure to uncheck both the `is_shared` box and the `is_global_default` box as a global default view will be automatically made as public to all users.
```

Comment:
Do we need a little JS so that unchecking `is_shared` automatically unchecks `is_global_default` and checking `is_global_default` automatically checks `is_shared`? Not blocking.

Reply:
Seeing this is not a required/urgent feature, I am going to open up a follow-up issue.

Reply:
https://github.com/nautobot/nautobot/issues/5827